### ISSUE TITLE
Unable to override most provisioner settings via suite 'attributes'

### ISSUE THREAD

**Issue by rafe-delphix** [neillturner/kitchen-ansible#156]
I regret that I find the documentation on this a little hard to follow.  In this section it says:

  https://github.com/neillturner/kitchen-ansible/blob/master/provisioner_options.md#configuring-provisioner-options

...It says "The provisioner can be configured globally or per suite"

  The example does not demonstrate how to configure a provisioner at the suite level as far as I can tell.

Then in this section:

  https://github.com/neillturner/kitchen-ansible/blob/master/provisioner_options.md#ruby-install-to-run-serverspec-verify

...it says "To override a setting at the suite-level, specify the setting name under the suite's attributes:" and shows an example of how to override 'extra_vars' (this example looks out of place - the section is talking about verifiers).  I am trying to override a different provisioner setting 'idempotency_test', however I do not seem to be able to.  When I look at the source code, the only reference I can find to 'attributes' is:

  https://github.com/neillturner/kitchen-ansible/blob/f12981ac247f5d9f8f5ac653f7d7c4f706b06da6/lib/kitchen/provisioner/ansible_playbook.rb#L673

It seems that the values in attributes are only applicable to the provioner 'extra_vars'.  Is this intentional or should other provisioner values be overridable?

**Comment by rafe-delphix** [comment]
I'm retracting this bug report.  I misunderstood how kitchen sets provisioner variables, had not realized that you can provide a 'provisioner' attribute to a suite.  The kitchen documentation might have been clearer, but the solution is also intuitive.

**Comment by neillturner** [comment]
Thanks for logging the issue. it will help others using test kitchen. This area of suite variables is part of test-kitchen itself rather than the provisioner.  I will test and add an example of in my docs though.
